src/app/voting/add-edit/add-edit.component.css

```
@@ -0,0 +1,64 @@
+.wrapper-for-divs {
+  display: flex;
+}
+
+.great-box {
+  display: flex;
+  width: 200px;
+  height: 200px;
+  background-color: gray;
+}
+
+/*.alignleft {*/
+/*  float: left;*/
+/*  display: block;*/
+/*  text-align: left;*/
+/*  width: 48%;*/
+
+/*}*/
+
+.box-with-participants {
+  background-color: gray;
+  width: 200px;
+  height: 200px;
+}
+
+.two-col {
+  display: grid;
+  grid-template-columns: 1fr auto;
+  column-gap: 1rem;
+}
+
+.two-even-col {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  column-gap: 1rem;
+}
+
+.list-item {
+  border-radius: 20px !important;
+  background: #fdd516;
+  list-style: none;
+}
+
+.list-item {
+  margin-bottom: 1rem;
+}
+
+li {
+  padding: 0.75rem 1rem 0.75rem 1rem;
+}
+
+.center {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+.left-align {
+  display: flex;
+  flex-direction: column;
+  align-items: start;
+  justify-content: start;
+}
```

```suggestion

.box-with-participants {
  background-color: gray;
  width: 200px;
  height: 200px;
}

```

Remove duplicate rules.

---

can you help with the styling issues if you have time?

---

@bognern